streams/src/main/java/org/apache/kafka/streams/kstream/internals/Change.java

```diff
@@ -22,10 +22,16 @@
 
     public final T newValue;
     public final T oldValue;
+    public final boolean isLatest;
 
     public Change(final T newValue, final T oldValue) {
+        this(newValue, oldValue, true);
```

Comment:
This two-arg constructor is currently still being called from places such as the windowed aggregate processors and caching state stores, since versioned tables do not come into play here and therefore all Changes are considered the latest. If we think it'd be better to encode true in these places and remove the two-arg constructor, I can do this cleanup in a follow-up PR.

Reply:
I think it's fine